tools/scripts/print-commits.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Get the current branch name
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+# Get the most recent tag by commit history (linear, not just nearest)
+MOST_RECENT_TAG=$(git tag --sort=-creatordate | head -n 1)
+
+# Check if a tag was found
+if [ -z "$MOST_RECENT_TAG" ]; then
+  echo "No tags found in the repository."
+  exit 1
+fi
+
+# Print commit messages, hashes, and timestamps between the most recent tag and the current branch
+echo "Commit messages between branch '$CURRENT_BRANCH' and the most recent tag '$MOST_RECENT_TAG':"
+git log --pretty=format:"%cd - %an - %h - %s" --date=short $MOST_RECENT_TAG..$CURRENT_BRANCH
```

Comment:
This file is included in this release because it is a script I commonly use as part of my process for writing the `CHANGELOG.md` file, and I thought including it would be worthwhile. 

Please let me know if that is insufficient reason to include it as part of the release, and I will exclude the change.

Reply:
This ideally should be added in it's own PR that goes through a separate review and can be merged into `development`.

Reply:
10-4, I will break this out into its own PR.